open_academy/models/session.py

```diff
@@ -1,14 +1,48 @@
-from odoo import models, fields
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
 
 
 class Session(models.Model):
     _name = "session"
     _description = "Session"
 
     name = fields.Char(required=True)
-    start_date = fields.Date()
+    start_date = fields.Date(default=lambda self: fields.Date.today(self))
     duration = fields.Float()
     seats = fields.Integer(string="Number of seats")
+    taken_seats = fields.Integer()
```

Comment:
Remove `taken_seats`, instead of making the user update it manually use `len(attende_ids)` to know this number